core-interfaces/IBrowserEmulator.ts

```diff
@@ -1,10 +1,10 @@
 import INetworkInterceptorDelegate from './INetworkInterceptorDelegate';
 import IUserProfile from './IUserProfile';
-import IUserAgent from './IUserAgent';
 import INewDocumentInjectedScript from './INewDocumentInjectedScript';
 
 export default interface IBrowserEmulator {
-  readonly userAgent: IUserAgent;
+  readonly navigatorUserAgent: string;
```

Comment:
This user agent is used for network requests too right?

Reply:
Yes.

Reply:
Does the name navigator reflect that you were seeing it in window.navigator? Or are you thinking of "navigator" as the browser?

Reply:
It reflects window.navigator. I can see an argument both for/against it.